docs/01-getting-started.md

```diff
@@ -0,0 +1,43 @@
+# Getting started
+
+Getting started with balenaSound is as simple as deploying it to a [balenaCloud](https://balena.io/cloud) application; no additional configuration is required (unless you're using a DAC HAT).
+We've outlined the installation steps below, if you want an in depth step by step tutorial on how to get balenaSound up and running feel free to check this blog posts:
```

Comment:
Should be two sentences.

"We've outlined the installation steps below. If you want an in depth step by step tutorial on how to get balenaSound up and running feel free to check this blog posts:"